docs/controls/file-pickers/node/theming-customization.md

```diff
@@ -0,0 +1,89 @@
+---
+author: natalieethell
+ms.author: natalieethell
+ms.date: 10/03/2018
+ms.topic: conceptual
+title: Add custom theme to file browser component
+---
+
+# Theming customization
+
+You can give your File Browser a custom theme by utilizing Fabric's `Customizer` component.
+
+In your React app, import `Customizer` from `office-ui-fabric-react/lib/Utilities` and `ITheme` from `office-ui-fabric/lib/Styling`. Create a custom `ITheme` and pass it into the `settings` prop of `Customizer`. Finally, wrap `Customizer` around `GraphFileBrowser` to see your custom theme reflected in your component.
+
+The following example uses a theme pulled directly from [Fabric's theme generator](https://developer.microsoft.com/en-us/fabric#/styles/themegenerator) and passes it into an instance of Fabric's `createTheme` method.
```

Comment:
Perhaps we can add a

> as covered on [this page](https://developer.microsoft.com/en-us/fabric#/components/themes)

to the end of this paragraph or something similar.. tho the page is a bit sparse. Let's see what others say.

Reply:
Hmm, I agree that maybe we should add a little more detail on the theming methods. [This link](https://developer.microsoft.com/en-us/fabric#/components/themes) doesn't seem to cover `createTheme`, though -- maybe `createTheme` is something that should be added to that page?